openssl-sys/src/lib.rs

```diff
@@ -1,7 +1,9 @@
+#![feature(collections)]
```

Comment:
We can't use rustc features without giving up support for stable Rust distributions.